src/views/Login/Login.jsx

```diff
@@ -137,6 +138,7 @@ class Login extends React.Component {
   }
 
   render () {
+    const mfa = env.MultipleFactorAuthentication()
```

Comment:
I'm curious why there's a local `mfa` defined here on each render when there's an instance `this.mfa` created in the constructor (and `this.mfa` is never used 😛)

Reply:
Updated to use the local property